handbook/community.md

```diff
@@ -43,12 +43,12 @@ Keep headings brief and organize them in a logical order:
 Try to stay within 3 or 4 heading levels. Complicated documents may use more, but pages with a simpler structure are easier to read.
 
 ### Adding a link to the Fleet docs
-You can link documentation pages to each other using relative paths. For example, in `docs/01-Using-Fleet/01-Fleet-UI.md`, you can link to `docs/01-Using-Fleet/09-Permissions.md` by writing `[permissions](./09-Permissions.md)`. This will be automatically transformed into the appropriate URL for `fleetdm.com/docs`.
+You can link documentation pages to each other using relative paths. For example, in `docs/Using-Fleet/Fleet-UI.md`, you can link to `docs/Using-Fleet/Permissions.md` by writing `[permissions](./Permissions.md)`. This will be automatically transformed into the appropriate URL for `fleetdm.com/docs`.
```

Comment:
@eashaw Can we link to fleetdm.com here instead?  (ie. for links to docs from within the handbook?)

The auto-rewriting of doc links doesn't work properly for handbook, IIRC

Reply:
Also I'm pretty sure the Permissions.md link won't actually work regardless (check the path)

Reply:
Those two aren't actually links, they're [examples in the handbook](https://fleetdm.com/handbook/community#adding-a-link-to-the-fleet-docs).

Reply:
aHHHHHHHHHH sorry.  Fooled me!